service/routes.go

```diff
@@ -5,17 +5,18 @@ import (
 )
 
 func Routes(router *gin.Engine) *gin.Engine {
+	router.Use(middleware)
```

Comment:
not part of the `Routes` function it is part of the gin server setup..

Reply:
What do you mean? How is it different from the other routes?

Reply:
we will take it F2F on Sunday